tests/integration_tests/tests/agentless_tests/deployment_update/test_deployment_update_modification.py

```diff
@@ -540,3 +540,54 @@ def test_enable_workflow(self):
         assert cm.exception.error_code == 'unavailable_workflow_error'
         self._do_update(deployment.id, BLUEPRINT_ID)
         self.execute_workflow('wf1', deployment.id)  # doesn't throw
+
+    def test_workflows_availability_node_instances_state(self):
```

Comment:
I think I'd prefer to see this test not tied to update, sorry.

The other test is in here, because the `available` rule is strictly tied to update - updating the deployment is the only way to change that flag.

But for `node_instances_active`, I don't see a reason to make it related to update - I suggest instead of updating, just have wf1, wf2, and wf3 in the blueprint, and check them all directly.

It's also going to be a way faster test then

Reply:
:+1: